lib/unimatrix/authorization/version.rb

```diff
@@ -1,5 +1,5 @@
 module Unimatrix
   module Authorization
-    VERSION = "1.1.2"
+    VERSION = File.read( File.expand_path( '../../../../VERSION', __FILE__ ) ).chomp
```

Comment:
This is in the SDK though so may not have a Rails.root, since it will be used by workers and such right?